alkira/resource_alkira_connector_azure_vnet_helper.go

```diff
@@ -221,7 +221,10 @@ func constructVnetRouting(d *schema.ResourceData) (*alkira.ConnectorVnetRouting,
 		ExportOptions: exportOptions,
 		ImportOptions: importOptions,
 		ServiceRoutes: serviceRoutes,
-		UdrLists:      udrLists,
+	}
+
+	if len(udrLists.Subnets) > 0 || len(udrLists.Cidrs) > 0 {
```

Comment:
Can udrLists be nil?

Reply:
no, it is being initialized in line 87

Reply:
If this happens to UdrList, same thing will happen to ServiceRoutes, nope? If we choose to use this kind super nested arrays, this will be just impossible to maintain.